packages/apputils/src/thememanager.ts

```diff
@@ -360,4 +389,18 @@ namespace Private {
     each(widget.children(), fitAll);
     widget.fit();
   }
+
+  export function calcWidgetStyle(widget: Widget): void {
```

Comment:
When is this function used?

Reply:
It's not. That was from an earlier attempt at forcing a refresh that didn't quite work. Uggh. Sorry, I guess I forgot to delint  the PR while I was submitting it.

Reply:
done